gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -973,7 +972,8 @@ public static Builder builder() {
       private final Set<BlobSourceOption> sourceOptions = new LinkedHashSet<>();
       private final Set<BlobTargetOption> targetOptions = new LinkedHashSet<>();
       private BlobId source;
-      private BlobInfo target;
+      private BlobId targetId;
```

Comment:
Considering that this is private and we control how it is set I think it is fine, however my preference
would have been not to have duplicate `BlobId` (which conceptually can get out of sync) and instead have a boolean to indicate if body should be used to replace metadata